### PR TITLE
Create data models

### PR DESCRIPTION
**Pull request by Kwaku28**
## This part of the project is implemented to create data models

Tasks completed include the following:

1. Created tables for `User`, `Post`, `Comment`, and `Like`
2. Added associations to the model files

Thank you🤝

### CODE REVIEW

File: db/migrate/20230713045918_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :Name
+      t.string :Photo
+      t.string :Bio
+      t.integer :PostsCounter
```
Comment (elisha2kyakpo1) [inline db/migrate/20230713045918_create_users.rb]: - [ ] Kindly follow Ruby convention for data attributes, `Name` should be `name`, `Photo` should be `photo`, `Bio` should be `bio`, `PostsCounter` should be `posts_counter`

File: db/migrate/20230713050201_create_posts.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :Title
+      t.text :Text
+      t.integer :CommentsCounter
+      t.integer :LikesCounter
```
Comment (elisha2kyakpo1) [inline db/migrate/20230713050201_create_posts.rb]: - [ ] Kindly follow Ruby convention for data attributes, `Title` should be `title`, `Text` should be `text`, `CommentsCouter` should be `comments_counter`, `LikesCounter` should be `likes_counter`

File: db/migrate/20230713050401_create_comments.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :Text
```
Comment (elisha2kyakpo1) [inline db/migrate/20230713050401_create_comments.rb]: - [ ] Kindly follow Ruby convention for data attributes, `Title` should be `title`.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @Kwaku28, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description are good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.
✔️ Foreign keys are set up nicely👏
👏 The app works as expected.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20230713045918_create_users.rb (hunk 1 above)
Comment (Kwaku28) [reply db/migrate/20230713045918_create_users.rb]: Attributes updated

File: db/migrate/20230713050201_create_posts.rb (hunk 2 above)
Comment (Kwaku28) [reply db/migrate/20230713050201_create_posts.rb]: Attributes updated

File: db/migrate/20230713050401_create_comments.rb (hunk 3 above)
Comment (Kwaku28) [reply db/migrate/20230713050401_create_comments.rb]: Attributes updated

**Review by OLIPLICHE — APPROVED** [review]
Hi @Kwaku28, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- Applied the changes from the previous review👏
- Table and column names match the ERD diagram👏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**